style.css

```diff
@@ -68,3 +74,158 @@ main {
   color: #505f79;
   padding: 20px 20px 0 0;
 }
+
+.projects-portfolio {
+  display: grid;
+  justify-content: center;
+  border-radius: 16px;
+  width: 85%;
+  margin: 100px 0 0 28px;
+  height: auto;
+  padding-bottom: 114px;
+}
+
+.project-card {
+  display: flex;
+  flex-direction: column;
+  padding: 5%;
+  background: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+}
+
+.card-margin {
+  margin-top: 100px;
+}
+
+.project-detail h3 {
+  font-size: 30px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-bottom: 0;
+  width: 250px;
+}
+
+.role {
+  display: flex;
+  justify-content: space-between;
+  padding-left: 0;
+  padding-right: 2%;
+}
+
+.role li {
+  color: #7a869a;
+  font-size: 15px;
+}
+
+.role a {
+  color: #344563;
+  font-weight: 600;
+}
+
+.project-detail p {
+  color: #344563;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.skills {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  padding-left: 0;
+  font-size: 12px;
+}
+
+.skills li {
+  color: #6070ff;
+  padding: 4px 12px;
+  line-height: 16px;
+  background: #ebebff;
+  border-radius: 8px;
+  margin-right: 8px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  font-weight: bold;
+}
+
+.btn {
+  margin-top: 12px;
+  padding: 12px;
+  background: #fff;
+  color: #396df2;
+  font-size: 17px;
+  border-radius: 8px;
+  border: 1px solid #396df2;
+  font-weight: 600;
+}
+
+.about-container {
+  background: #e5e5e5;
+}
+
+.about {
+  background: #fff;
+  border-top-right-radius: 100px;
+  padding: 24px;
+}
+
+.about h3 {
+  margin-top: 90px;
+  margin-bottom: 0;
+  font-size: 38px;
+  line-height: 52px;
+  color: #172b4d;
+  font-weight: 800;
+}
+
+.dropdown {
+  padding-left: 0;
+  margin-right: 58px;
+}
+
```

Comment:
- [ ] Please 🙏 adjust the `margin-right `of each technology tag to make it look like the figma design template. I suggest decreasing the margin to ` margin-right: 8px;` or around, as it seems more appropriate. And then adjusting padding-right on `.open` and `.close` to be `padding: 1rem;`, that would adjust icons placement 👍 

Your design:

<img width="358" alt="Screenshot 2023-03-25 at 8 13 02 AM" src="https://user-images.githubusercontent.com/44798044/227689009-226c16b7-f92f-4ea9-80b4-38b5526d7781.png">

Figma Design:

<img width="206" alt="Screenshot 2023-03-25 at 7 34 03 AM" src="https://user-images.githubusercontent.com/44798044/227689003-c51ba54c-78e3-4878-8308-6380e2cc559b.png">